package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@storybook/addon-styling-webpack",
-  "version": "1.0.1",
+  "version": "2.0.0",
```

Comment:
Please note: I've bumped the version as I saw Shaun was doing it himself and I suspect it's not auto-managed. If version is managed via release commits (using Shaun's identity :grimacing:), this will need to be reverted.